examples/async_peg_insert_drq/record_demo.py

```diff
@@ -49,8 +49,10 @@
         raise PermissionError(f"No permission to write to {file_dir}")
 
     while success_count < success_needed:
-        next_obs, rew, done, truncated, info = env.step(action=np.zeros((6,)))
-        actions = info["intervene_action"]
+        actions = np.zeros((6,))
+        next_obs, rew, done, truncated, info = env.step(action=actions)
+        if "intervene_action" in info:
+            actions = info["intervene_action"]
```

Comment:
how does this code run correctly without issues previously? 

if "intervene_action" is None, the previous code always assumes 
`actions = info["intervene_action"]`

and this doesn't throw out errors?

Reply:
My bad, there was a change in the IntervenetionWrapper that I missed. Now it puts `intervene_action ` in info only if  intervene_action is nonzero

Reply:
i was more talking the code in the left

```
        next_obs, rew, done, truncated, info = env.step(action=np.zeros((7,)))
        actions = info["intervene_action"]
```

does this code run without issues previously? the actions can be unassigned in certain cases

Reply:
it used to be always assigned. If no intervention was provided, then the policy action is just put in info['intervene_action'], but this meant the intervene_action were in two different frames depending on whether it's from the policy or the spacemouse